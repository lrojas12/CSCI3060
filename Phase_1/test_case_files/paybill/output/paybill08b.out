Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information
Mode in which you wish to login as:
Account holder’s name:
You are currently logged in as acc_holder
Bank Account Number: acc_num
Balance: balance
Transaction Payment Plan: plan
Status: status
Enter your account number:
Enter company:
ERROR: The company name is invalid
Must be one of the following: EC, CQ or TV 
You have successfully logged out of your account
