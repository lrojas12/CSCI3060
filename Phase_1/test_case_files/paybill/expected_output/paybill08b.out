Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
Enter account holder’s name:
You are currently logged in as Harley
Bank Account Number: 00018
Balance: $1000.00
Transaction Payment Plan: non-student
Status: active
Enter your account number:
Enter company:
Enter amount to pay:
ERROR:The amount entered is not valid.
You have successfully logged out of your account.